frontend/src/metabase/collections/components/BaseTableItem.jsx

```diff
@@ -59,47 +57,102 @@ export function BaseTableItem({
   const handleCopy = useCallback(() => onCopy([item]), [item, onCopy]);
   const handleArchive = useCallback(() => item.setArchived(true), [item]);
 
-  const testId = isPinned ? "pinned-collection-entry" : "collection-entry";
+  const renderRow = useCallback(() => {
+    const canSelect = typeof onToggleSelected === "function";
+
+    const lastEditInfo = item["last-edit-info"];
+
+    // We don't keep last edit info for pulses
+    // TODO Remove ternary when Pulses are gone
+    const lastEditedBy = lastEditInfo
+      ? `${lastEditInfo.first_name} ${lastEditInfo.last_name}`
+      : "";
+    const lastEditedAt = lastEditInfo
+      ? moment(lastEditInfo.timestamp).format("MMMM DD, YYYY")
+      : "";
+
+    const testId = isPinned ? "pinned-collection-entry" : "collection-entry";
+
+    return (
+      <tr
+        key={item.id}
+        data-testid={testId}
+        style={{
+          // Table row can be wrapped with ItemDragSource,
+          // that only accepts native DOM elements as its children
+          // So styled-components can't be used here
+
+          height: "80px",
+          borderBottom: hasBottomBorder ? `1px solid ${color("border")}` : "",
+        }}
+      >
```

Comment:
[not major] 

Looks like this style block is one of the most defensible situations to use multiline props inside a component call.

In many other places in the codebase, however, we have multiline props that really makes the code hard to chunk mentally, and understand fast.

One [random example here](https://github.com/metabase/metabase/blob/f4ab182d4cddc07c56efe4551aa492c25a04a4ae/frontend/src/metabase/parameters/components/ParameterTargetWidget.jsx#L52-L70): note how there are two props, one in a parent element, one in a child, with multiple lines. There are some of these with 7 or more lines.

See if you'd agree that a future reader of the block below would more quickly grasp what the `<tr/>` component is about, and if interested in updating the style, it's still easy to find and change.

Again, this suggestion is not so much about _this_ instantiation :), rather about all the others in the codebase.

```suggestion
    const testId = isPinned ? "pinned-collection-entry" : "collection-entry";
    
    const trStyles = {
      height: 80,
      borderBottom: hasBottomBorder ? `1px solid ${color("border")}` : "",
    }
    
    // Table row can be wrapped with ItemDragSource,
    // that only accepts native DOM elements as its children
    // So styled-components can't be used here
    return (
      <tr
        key={item.id}
        data-testid={testId}
        style={trStyles}
      >
```

Reply:
100% agree. I'm a big fan of not inlining props the way you suggest, I'm trying to follow the approach we have in the codebase